### PR TITLE
Add section for adding datasource through proposal

### PR DESCRIPTION
**Pull request by rajkharvar**

### CODE REVIEW

**Review by SkandaBhat — APPROVED** [review]

File: docs/Governance.md [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
+It is quite important to whitelist or backlist the data sources to prevent any misinformation and DoS attacks.
 
 The Razor Network governance portal can be accessed [here](https://snapshot.org/#/raz.eth). You need to hold $RAZOR tokens on Ethereum/Polygon Mainnet or $sRZR (staked $RAZOR tokens) on Razor SKALE Chain to be able to vote.
 
-**Note** - _Updated governance parameters can be checked [here](https://razorscan.io/governance/values)._
+## Add datasource through proposal
+
+1. Please begin by visiting the [datasources](https://github.com/razor-network/datasources) repository on GitHub. Once there, proceed to fork the repository to your own GitHub account.
+
+2. Next, make the necessary changes to the forked repository, incorporating the required information based on the `contributing.md` guidelines and project requirements. Once you have completed the changes, it's time to create a pull request (PR) to the 'master' branch of the original repository.
```
Comment (SkandaBhat) [inline docs/Governance.md:16]: Lnk the contributing.md here.

**Review by SkandaBhat — CHANGES_REQUESTED** [review]
link contrib link in the governance docs.

File: docs/Governance.md (hunk 1 above)
Comment (rajkharvar) [reply docs/Governance.md]: Resolved in [9597677](https://github.com/razor-network/docs-v2/pull/98/commits/95976779341b3f092b4245a4b20c5ad5afa381bf)

File: docs/Governance.md [hunk 2]
```diff
@@ -1,15 +1,27 @@
 # Governance
 
-Governance aims to enable the oracle network to do subjective decision making based on acceptions and rejections of the proposals. The proposals will be mostly around
+Governance aims to enable the oracle network to do subjective decision-making based on acceptions and rejections of the proposals. The proposals will be mostly around
 
-1. Acception or Rejection of Data soruces collections.
+1. Acception or Rejection of Data sources collections.
```
Comment (yohanelly95) [inline docs/Governance.md:5]: Acceptance* small typo @rajkharvar

File: docs/Governance.md [hunk 3]
```diff
@@ -1,15 +1,27 @@
...
 
-1. Acception or Rejection of Data soruces collections.
+1. Acceptance or Rejection of Data sources collections.
 2. Change in the parameters of the Validation layer.
 
-It is quite important to whitelist of backlist the data sources to prevent any misinformation and DOS attacks.
+It is quite important to whitelist or backlist the data sources to prevent any misinformation and DoS attacks.
 
-The Razor Network governance portal can be accessed [here](https://snapshot.org/#/raz.eth). You need to hold $RAZOR tokens on Ethereum/Polygon Mainnet or $sRZR (staked $RAZOR tokens) on Razor SKALE Chain to be able to vote.
+The Razor Network governance portal can be accessed [here](https://vote.razor.network/). You need to hold $RAZOR tokens on Ethereum/Polygon Mainnet or $sRZR (staked $RAZOR tokens) on Razor SKALE Chain to be able to vote. As a delegator on the Razor Network, it's essential to note that you are responsible for casting your governance votes. Currently, stakers to whom you've delegated cannot vote on your behalf.
```
Comment (SkandaBhat) [inline docs/Governance.md:10]: Add it in a new line starting with "NOTE:..."

File: docs/Governance.md [hunk 4]
```diff
@@ -1,15 +1,29 @@
...
 
-Governance aims to enable the oracle network to do subjective decision making based on acceptions and rejections of the proposals. The proposals will be mostly around
+Governance aims to enable the oracle network to do subjective decision-making based on acceptions and rejections of the proposals. The proposals will be mostly around
 
-1. Acception or Rejection of Data soruces collections.
+1. Acceptance or Rejection of Data sources collections.
 2. Change in the parameters of the Validation layer.
 
-It is quite important to whitelist of backlist the data sources to prevent any misinformation and DOS attacks.
+It is quite important to whitelist or backlist the data sources to prevent any misinformation and DoS attacks.
```
Comment (hrishikeshio) [inline docs/Governance.md:8]: make it add or remove data sources. we dont have blacklist.

Comment (rajkharvar) [reply docs/Governance.md]: Resolved [22d444b](https://github.com/razor-network/docs-v2/pull/98/commits/22d444b5b3e609169a7046b19b24e12ee63bc153)

**Pull request merged**